x-pack/plugins/infra/server/lib/alerting/common/utils.ts

```diff
@@ -153,6 +146,27 @@ export const getViewInInventoryAppUrl = ({
 export const getViewInMetricsAppUrl = (basePath: IBasePath, spaceId: string) =>
   addSpaceIdToPath(basePath.publicBaseUrl, spaceId, LINK_TO_METRICS_EXPLORER);
 
+export const getAlertUrl = async (
+  alertUuid: string | null,
+  spaceId: string,
+  startedAt: string,
+  alertsLocator?: LocatorPublic<AlertsLocatorParams>,
+  publicBaseUrl?: string
+) => {
+  if (!publicBaseUrl || !alertsLocator || !alertUuid) return '';
+
+  const rangeFrom = moment(startedAt).subtract('5', 'minute').toISOString();
```

Comment:
We probably want to take the window / look-back into account. Or perhaps rule interval.  And consider multiples of those - for example, the range should be 3 x window, or something.  If the look-back was an hour, or a day, seeing 5m may not be useful.

We'll probably get additional requirements for this anyway, so deferring till then is probably fine.

Reply:
Good point, I was also thinking about the same but wanted to have smaller PRs focused on generating the link for infra rules for now.
In another instance, I used 1/5 of the alert duration period before the start of the alert and 1/5 after the alert ends (in case it was recovered.)

There is room for improvement, and I created the following ticket to improve this time range:
https://github.com/elastic/kibana/issues/158480